brig/cmd/brig/commands/dashboard.go

```diff
@@ -0,0 +1,99 @@
+package commands
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/Azure/brigade/pkg/portforwarder"
+	"github.com/bacongobbler/browser"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	dashboardUsage = `Opens the Kashti dashboard with the default browser.
+`
+	remoteAPIPort = 7745
+	remotePort    = 80
+)
+
+var (
+	port            int
+	apiPort         int
+	kashtiNamespace string
+	openDashboard   bool
+)
+
+func init() {
+	Root.AddCommand(dashboard)
+
+	flags := dashboard.PersistentFlags()
+	flags.IntVar(&port, "port", 8081, "local port for the Kashti dashboard")
+	flags.IntVar(&apiPort, "api-port", 7745, "local port for the Brigade API server")
+	flags.StringVar(&kashtiNamespace, "kashti-namespace", "default", "namespace for Kashti")
+	flags.BoolVar(&openDashboard, "open-dashboard", true, "open the dashboard in the browser")
```

Comment:
Should the default be `true` here?
For example this always fails on Bash on Windows (WSL) as `xdg-utils` is not installed by default. (an manually adding it with `apt-get` still won't work). This means the command always panics.

Reply:
oh good point. I forgot about WSL. I'll see if there's a way we can detect that we're running under WSL. It's a bug in https://github.com/bacongobbler/browser

@technosophos any thoughts on flipping this flag off by default?

Reply:
I've done searched about detecting WSL and it should not be that difficult. 

From WSL:

```
$ cat /proc/sys/kernel/osrelease
4.4.0-17134-Microsoft

$ cat /proc/version
Linux version 4.4.0-17134-Microsoft (Microsoft@Microsoft.com) (gcc version 5.4.0 (GCC) ) #137-Microsoft Thu Jun 14 18:46:00 PST 2018

$ uname -a
Linux <PC-name> 4.4.0-17134-Microsoft #137-Microsoft Thu Jun 14 18:46:00 PST 2018 x86_64 x86_64 x86_64 GNU/Linux
```

So checking either of those should tell us if running under WSL or not.

I'm not sure the default should be `false` though - if we detect it's WSL and output that we cannot open the browser, then everybody else (which I think is a bigger number) can still benefit from the default.

What do you think?

Reply:
Yeah I think we can special-case WSL. We might even be able to detect we're running under WSL and call `cmd /c start` from within WSL. I'll investigate

Reply:
https://github.com/bacongobbler/browser/issues/3

Reply:
Nice! I think I like your solution more!
Should this case be treated by `bacongobbler/browser`?